### PR TITLE
Added sample code - JDBC 21c Driver Support for Virtual Threads

### PR DESCRIPTION
**Pull request by juarezjuniorgithub**
@Kuassim 
Signed-off-by: Juarez Barbosa Junior juarez.barbosa@oracle.com

### CODE REVIEW

File: java/jdbcdriver-virtual-threads/src/main/java/com/oracle/dev/jdbc/DatabaseConfig.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * </p>
+ */
+public class DatabaseConfig {
+
+	private static final Properties CONFIG = new Properties();
+
+	static {
+		try {
+			var fileStream = Files.newInputStream(
+					Path.of("C:\\java-projects\\jdbcdriver-virtual-threads\\src\\main\\resources\\config.properties"));
```
Comment (Kuassim) [inline java/jdbcdriver-virtual-threads/src/main/java/com/oracle/dev/jdbc/DatabaseConfig.java:41]: Can you also add a Linux path?

File: java/jdbcdriver-virtual-threads/src/main/java/com/oracle/dev/jdbc/PlatformThreadsOracleJdbc.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+		Properties info = new Properties();
+		info.put(OracleConnection.CONNECTION_PROPERTY_USER_NAME, DB_USER);
+		info.put(OracleConnection.CONNECTION_PROPERTY_PASSWORD, DB_PASSWORD);
+		info.put(OracleConnection.CONNECTION_PROPERTY_FAN_ENABLED, false);
+		OracleDataSource ods = new OracleDataSource();
+		ods.setURL(DB_URL);
+		ods.setConnectionProperties(info);
+
+		OracleConnection connection = (OracleConnection) ods.getConnection();
```
Comment (Kuassim) [inline java/jdbcdriver-virtual-threads/src/main/java/com/oracle/dev/jdbc/PlatformThreadsOracleJdbc.java:54]: Would be nice to show pooled connection with UCP.
PoolDataSource pds = PoolDataSourceFactory.getPoolDataSource();

**Review by jeandelavarene — DISMISSED** [review]

**Review by gvenzl — CHANGES_REQUESTED** [review]
Please address the comments raised by @Kuassim

File: java/jdbcdriver-virtual-threads/src/main/java/com/oracle/dev/jdbc/PlatformThreadsOracleJdbc.java (hunk 2 above)
Comment (juarezjuniorgithub) [reply java/jdbcdriver-virtual-threads/src/main/java/com/oracle/dev/jdbc/PlatformThreadsOracleJdbc.java:54]: @Kuassim done - added pooled connection with UCP for both examples (platform threads, virtual threads).

File: java/jdbcdriver-virtual-threads/src/main/java/com/oracle/dev/jdbc/DatabaseConfig.java (hunk 1 above)
Comment (juarezjuniorgithub) [reply java/jdbcdriver-virtual-threads/src/main/java/com/oracle/dev/jdbc/DatabaseConfig.java]: @Kuassim I have improved this one as well (removed a dependency on absolute paths).

**Comment by juarezjuniorgithub** [comment]
@Kuassim @gvenzl can you review again and approve? Thanks in advance!

**Review by Kuassim — APPROVED** [review]

**Review by gvenzl — APPROVED** [review]

**Pull request merged**